cysync/src/store/hooks/flows/useCardAuth.ts

```diff
@@ -109,6 +112,15 @@ export const useCardAuth: UseCardAuth = isInitial => {
     setIsInFlow,
     deviceState
   } = useConnection();
+  const { connected: internetConnected } = useNetwork();
+
+  useEffect(() => {
+    setEnableRetry(!!deviceConnection && internetConnected);
+  }, [deviceConnection, internetConnected]);
+
+  useEffect(() => {
+    if (errorObj.isSet && internetConnected) setShowRetry(true);
```

Comment:
This should also depend on errorObj, right?

Reply:
Yes yes, adding it to the dependency list